### ISSUE TITLE
Remove unwanted classes using pre-trained weights 

### ISSUE THREAD

**Issue by lisa676** [ultralytics/yolov5#1025]
How to remove unwanted classes using pre-trained weights

I want to detect only persons from the pre-trained weights. I don't want to train the model again.
How to remove others labels and classes?

**Comment by glenn-jocher** [comment]
@lan786 if you want to use pretrained weights for person detection only:
`python detect.py --classes 0`

**Comment by lisa676** [comment]
@glenn-jocher Thanks for your help
